NSS-Spontaneous-Spoonbills/Sprint2
# Em computer testing

## Link to Ticket

Closes #8

## Description of Proposed Changes

* Added testing for computers

## Steps to Test

1. Start virtual environment and server
1. You shouldn't need to rerun migrations because the test examples are built in
1. Run the following in the console: ```python manage.py test```
1. All tests should return OK

```sh
git fetch --all
git checkout em_computer_testing
```

## Impacted Areas in Application

**New**

WorkforceManagement/test/test_computers.py
WorkforceManagement/test/test_models/py

**Modified**

WorkforceManagement/urls.py

## Mentions @username

@obscure-shadow @JacobAndrewSmith92 @CashewRose @binauralbeat @kimberly-bird 

## Testing


## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
